doc/vision/vision.md

```diff
@@ -0,0 +1,101 @@
+# Vision
+The Research Paper Analyser examines to what extent research work is related to each other, 
+i.e. whether and how often it refers to each other. 
+For example, if paper C is based on paper B and paper B is based on paper A, then papers A, B and C are connected. 
```

Comment:
Clarify: 
```
A->B->C
```
or
```
A->B->C
`-----^
```
I would strongly argue for the first one (e.g. no transitive dependencies) as this would add too much clutter to the graph.